src/main/java/nl/tudelft/ti2206/group9/level/State.java

```diff
@@ -65,7 +66,7 @@ public static void resetAll() {
         setCoins(0);
         highscore = 0;
         previousDistance = 0;
-        skin = Skin.getNoob();
+        skin = Skin.loadSkinsToList().get(0);
```

Comment:
Yes, it is true that i hardcoded this. But isn't it logical the first item on the list is the noobskin? I wanted to get rid of all the getters and this is the best way I imagined.